src/Button/DigitizeButton/DigitizeButton.example.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: basic.hbs
+title: DigitizeButton example
+description: This is a example showing digitize buttons with different interactions.
```

Comment:
*an*

Reply:
Thanks for reviews guys :pineapple: 

Everything is fixed, I'll merge now